client/my-sites/importer/author-mapping-pane.jsx

```diff
@@ -46,7 +46,7 @@ class AuthorMappingPane extends PureComponent {
 		if ( numTargetUsers === 1 && numSourceUsers === 1 ) {
 			return this.props.translate(
 				'There is one author on your %(sourceType)s site. ' +
-					"Because you're the only author on {{b}}%(destinationSiteTitle)s{{/b}}, " +
+					"Because you're the only author on your new site, " +
```

Comment:
Can we use "this site (**foo.wordpress.com**)" to be more specific?
```suggestion
					"Because you're the only author on this site ({{b}}%(siteSlug)s{{/b}}), " +
```